civicaseextras.civix.php

```diff
@@ -486,7 +486,20 @@ function _civicaseextras_civix_civicrm_entityTypes(&$entityTypes) {
   ));
 }
 
+/**
+ * (Delegated) Implements hook_civicrm_alterAngular().
+ */
 function _civicaseextras_civicrm_alterAngular(\Civi\Angular\Manager &$angular) {
+  _civicaseextras_civicrm_alterAngular__addCaseOutcomesPanel($angular);
+  _civicaseextras_civicrm_alterAngular__addCaseDurationToCaseList($angular);
+}
+
+/**
+ * Adds a case outcomes panel to the case details.
+ *
+ * @param Manager $angular as provided by the alter angular hook.
+ */
+function _civicaseextras_civicrm_alterAngular__addCaseOutcomesPanel(\Civi\Angular\Manager &$angular) {
```

Comment:
Just to make the code a bit shorter can you add `use` statements for namespaces classes at the top of the file and just use their shortname instead?